.github/workflows/check.yml

```diff
@@ -87,7 +87,17 @@ jobs:
       - name: Sqllogictest
         run: |
           psql -f ./tests/init.sql
-          sqllogictest -u runner -d runner './tests/**/*.slt'
+          sqllogictest -u runner -d runner './tests/sqllogictest/*.slt'
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10' 
+      - name: Crash Recovery test
+        run: |
+          sudo pip install psutil==5.9.7
+          psql -f ./tests/init.sql
+          sqllogictest -u runner -d runner './tests/crash/create.slt'
+          sudo python ./tests/crash/kill.py
```

Comment:
Why does python/pip require `sudo`?

Reply:
Kill process need a priviledge, and sudo python would use a seperated interpreter from local user. I understand sudo is soso, if there is a better way?

Reply:
It's necessary if the process is started with `sudo`.